env/work/srw_python/SRWLIB_Example15.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
```

Comment:
SRWLIB_Example15.py should be modified to implement saving some of (or all) result to files (into the folder data_example_15), so that even if Matplotlib is not installed, the example would still be runnable and the results readable, as this is the case for all examples SRWLIB_Example01.py - SRWLIB_Example14.py.